src/Tools/Console/Command/MappingDescribeCommand.php

```diff
@@ -52,27 +53,44 @@ final class MappingDescribeCommand extends AbstractEntityManagerCommand
     protected function configure(): void
     {
         $this->setName('orm:mapping:describe')
-             ->addArgument('entityName', InputArgument::REQUIRED, 'Full or partial name of entity')
-             ->setDescription('Display information about mapped objects')
-             ->addOption('em', null, InputOption::VALUE_REQUIRED, 'Name of the entity manager to operate on')
-             ->setHelp(<<<'EOT'
+            ->addArgument('entityName', InputArgument::REQUIRED, 'Full or partial name of entity')
+            ->setDescription('Display information about mapped objects')
+            ->addOption('em', null, InputOption::VALUE_REQUIRED, 'Name of the entity manager to operate on')
+            ->addOption(
+                'format',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Output format (text, json)',
+                'text',
```

Comment:
```suggestion
               MappingDescribeCommandFormat::TEXT,
```

`TEXT` is currently not used, so this would be a good place to introduce it.

Reply:
fair point, done :)